ejerciciosPilas/src/actores/ActorAnimado.ts

```diff
@@ -38,6 +39,24 @@ class ActorAnimado extends Actor {
         this.pilaAnimaciones = [];
     }
 
+    acelerarLaVelocidadDeLasAnimaciones() {
+        // TODO: pasar esto a pilasweb (como minimo a la _imagen)
+        this.velocidadOriginalDeLasAnimaciones = {};
+        for (var nombre in this._imagen.animaciones) {
+            this.velocidadOriginalDeLasAnimaciones[nombre] = this._imagen.animaciones[nombre].velocidad;
```

Comment:
No entiendo este for: `nombre` es un string? `this._imagen.animaciones` es una colección de strings? Si es así por qué se hace `this._imagen.animaciones[nombre].velocidad`? Y si hay un objeto animación, no se puede trabajar directamente con él?

Reply:
@51lv3str1 puede ser que hayamos olvidado eliminar estos métodos?

Reply:
Che, si, esto esta en pilas web, mañana lo vuelo